src/units/store.rs

```diff
@@ -99,28 +70,9 @@ impl<'a, H: Hasher, D: Data, KB: KeyBox> UnitStore<'a, H, D, KB> {
             warn!(target: "AlephBFT-unit-store", "Trying to mark the node {:?} as forker for the second time.", forker);
         }
         self.is_forker[forker] = true;
-        let forkers_units = (0..=self.round_in_progress)
+        (0..=self.max_round)
```

Comment:
Since `round_in_progress` is no longer there, update the comment above this function.